### ISSUE TITLE
Add hint for missing alt text in media preview

### ISSUE THREAD

**Issue by leahoswald** [mastodon/mastodon#23535]
### Pitch

Some users don't want to boost images that are lacking a media description. Currently you can't see this without hovering over the image. Some clients already show if there is a description or not. I would suggest to add the code created by @janw for toot.haus [in this commit](https://github.com/toothaus/mastodon/commit/fa6382acebc786d9e33027268e34aad6c870d52b) to the mastodon base code. I could be an option that can be activated by the user in their settings.

<img width="631" alt="d128df91e2bde1ac" src="https://user-images.githubusercontent.com/1292551/218272569-8b274bf2-db51-4412-b0d3-4ab61592680e.png">

### Motivation

This is a feature to help make the fediverse and mastodon more friendly for users who are visually impaired and those who want to support them.

**Comment by jdebp** [comment]
See also https://blogs.toot.wales/toot/alt-text-february and https://raw.githubusercontent.com/jazmichaelking/mastodon-single-column-css/master/fuzzy.css .

**Comment by janw** [comment]
Thank you @leahoswald for opening this.

If there is an interest in this feature by the maintainers, I'd be happy to create a corresponding PR. 🙏
